### PR TITLE
Integrate block cache

### PR DESCRIPTION
**Pull request by khenderick**
Note: due to time restrictions, this build is not intended for OSE installs.

### CODE REVIEW

File: webapps/frontend/app/viewmodels/wizards/addvpool/data.js [hunk 1]
```diff
@@ -79,42 +79,54 @@ define(['jquery', 'knockout'], function($, ko){
 
     singleton = function() {
         var wizardData = {
-            // 'aa' stands for Accelerated Alba
+            // 'FC' stands for Accelerated Alba
```
Comment (kvanhijf) [inline webapps/frontend/app/viewmodels/wizards/addvpool/data.js]: FC Stands for Accelerated ALBA??

File: webapps/frontend/app/viewmodels/wizards/addvpool/gather_block_cache.js [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+        self.data   = data;
+        self.shared = shared;
+
+        // Handles
+        self.fetchAlbaVPoolHandle = undefined;
+
+        // Observables
+        self.albaPresetMap          = ko.observable({});
+        self.backendsBC             = ko.observableArray([]);
+        self.blockCacheSettings  = ko.observableArray(['write', 'read', 'rw', 'none']);
```
Comment (kvanhijf) [inline webapps/frontend/app/viewmodels/wizards/addvpool/gather_block_cache.js]: Outlining

File: webapps/frontend/app/viewmodels/wizards/addvpool/gather_fragment_cache.js [hunk 3]
```diff
@@ -57,7 +57,7 @@ define([
                 return temp;
             }
             $.each(self.data.storageRoutersUsed(), function(index, sr) {
-                if (self.data.vPool().metadata().hasOwnProperty('backend_aa_' + sr.guid())) {
+                if (self.data.vPool().metadata().hasOwnProperty('backend_FC_' + sr.guid())) {
```
Comment (kvanhijf) [inline webapps/frontend/app/viewmodels/wizards/addvpool/gather_fragment_cache.js]: Should be lowercase, perhaps check gather_block_cache too

**Review by kvanhijf — CHANGES_REQUESTED** [review]

**Comment by khenderick** [comment]
Going to test a few more scenarios:
* StorageRouter reuse
* Extend/shrink vPool
* Behaviour with no EE

**Review by kvanhijf — APPROVED** [review]

**Pull request merged**